crates/bevy_image/Cargo.toml

```diff
@@ -49,6 +49,7 @@ image = { version = "0.25.2", default-features = false }
 # misc
 bitflags = { version = "2.3", features = ["serde"] }
 bytemuck = { version = "1.5" }
+wgpu-types = { version = "23", default-features = false }
 wgpu = { version = "23.0.1", default-features = false }
```

Comment:
Reminder to remember to remove this

Reply:
can't remove it this PR, but once we upgrade wgpu we can